src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,58 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public int splitAndSum(String value)throws Exception{
+        int result = checkNull_Empty(value);
+        String[] values;
+
+        if(result == 0){
+            return result;
+        }else{
+            values = checkSeparator(value);
+            int calcul = 0;
+            for(String val : values){
+                calcul += checkNegative(Integer.valueOf(val));
+            }
+            result = calcul;
+        }
+        return result;
+    }
+
+    public int checkNull_Empty(String value)throws Exception{
+        int result = -1;
+
+        if(value == null){
+            result = 0;
+        }else if(value.isEmpty()){
+            result = 0;
+        }
+        return result;
```

Comment:
이번 미션을 진행하면서 if - else 구문을 사용하지 않고 구현해보는것을 경험해보세요 😄 
위 코드는 아래와 같이 변경할수 있을것 같아요 🙏 
```suggestion
        if(value == null || value.isEmpty()){
            result = 0;
        }
        return result;
```